### PR TITLE
ClosedLoop - BZ 2073030 - Support --yes-i-know flag on RHCS 6.0

### PR DESCRIPTION
**Pull request by sayaleeraut**
ClosedLoop - BZ 2073030 - Support --yes-i-know flag on RHCS 6.0

Signed-off-by: sayaleeraut <saraut@redhat.com>

# Description
ClosedLoop - BZ 2073030 - Support --yes-i-know flag on RHCS 6.0
Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [✓] Create a test case in Polarion reviewed and approved.
- [✓] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [✓] Review the automation design
- [ ] Implement the test script and perform test runs
- [✓] Submit PR for code review and approve
- [✓] Update Polarion Test with Automation script details and update automation fields
- [✓] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: ceph/ceph_admin/bootstrap.py [hunk 1]
```diff
@@ -275,7 +275,7 @@ def bootstrap(self: CephAdmProtocol, config: Dict):
 
         # Todo: This patch is specific to 5.1 release,
         #   should be removed for next 5.x development builds or release.
-        if rhbuild.split("-")[0] in ["5.1", "5.2"]:
+        if rhbuild.split("-")[0] in ["5.1", "5.2", "6.0"]:
```
Comment (psathyan) [inline ceph/ceph_admin/bootstrap.py:278]: We should not have the flag set for `6.0`... do you have information that mandates the need to use this flag.

**Review by psathyan — COMMENTED** [review]
Currently we don't have full fledged unit test suites hence we recommend attaching an execution log done with the changes.

File: ceph/ceph_admin/bootstrap.py (hunk 1 above)
Comment (sayaleeraut) [reply ceph/ceph_admin/bootstrap.py:278]: Hey @psathyan , I am adding this change for BZ https://bugzilla.redhat.com/show_bug.cgi?id=2073030
Post RHCS 6.0 release, when RGW multisite is supported, we will stop requiring deployments to pass --yes-i-know to 'cephadm bootstrap'. When this happens, the reuest in this BZ is to please do not make the deployment fail if --yes-i-know is passed.
The bug requests that --yes-i-know remain a recognized option, even if it doesn't do anything. Hence, I added this flag over here.
Please let me know if any more details are required.

Comment (sayaleeraut) [reply ceph/ceph_admin/bootstrap.py:278]: Okay, Thanks @psathyan

**Comment by psathyan** [comment]
IMO, we should have both options for `6.0`... could we revisit this PR?

**Comment by vdas-redhat** [comment]
This is pending re-testing with both options @sayaleeraut

**Comment by veera-raghava-reddy** [comment]
I would recommend not to use "--yes-i-know" flag for 6.0 and close this PR.
We can perform one-time manual verification of Bug https://bugzilla.redhat.com/show_bug.cgi?id=2073030

@sayaleeraut @vdas-redhat @psathyan

**Comment by vdas-redhat** [comment]
Closing this PR. Manual testing to be performed on this issue and if we feel a PR is a must we come back and submit a new one.

**Pull request closed without merging**